locales/it.yml

```diff
@@ -0,0 +1,12 @@
+---
+it:
+  header:
+    title: "Berlin Code of Conduct"
+    sub-title: "Un Codice di Condotta per tutti gli user group e le conferenze"
+  supporters:
+    title: "Aderenti"
+    sub-title: "Gli user group e le conferenze che supportano il Berlin Code of Conduct"
+    add: "Se vuoi aggiungere alla lista il tuo user group o la tua conferenza, semplicemente crea una pull request su %{link:our repository}."
+  footer:
+    attribution: "Questo Codice di Condotta e' adattato dal %{link:pdx.rb code of conduct}."
```

Comment:
I'll keep it short so it might look rude, but just imagine I add a "please" and a ❤️  after each comment :p

`è` instead of `e'`